### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by igor-stasiv**
[DEMO LINK](https://igor-stasiv.github.io/react_sum/)

### CODE REVIEW

**Review by Burunduk003 — APPROVED** [review]
Well done!

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
 
+import Sum from './components/Sum';
+
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <Sum
+      a={2}
+      b={2}
+    />
```
Comment (VinnieJ-2k20) [inline src/App.js:11]: No need to add new lines here, code is short enough.
```suggestion
    <Sum a={2} b={2} />
```

File: src/components/Sum.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <p>
+      Sum of
+      {a}
+      {' '}
+      and
+      {b}
+      {' '}
+      is
+      {a + b}
```
Comment (VinnieJ-2k20) [inline src/components/Sum.js:14]: There is much easier way to render this text:
```suggestion
  {`Sum of ${a} and ${b} is ${a + b}`}
```

**Review by VinnieJ-2k20 — CHANGES_REQUESTED** [review]
Good solution.
Let's make some final changes.

File: src/components/Sum.js (hunk 2 above)
Comment (igor-stasiv) [reply src/components/Sum.js:14]: It is automatically changed by linter and I don't know how to fix it

**Review by VinnieJ-2k20 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**